tests/processes/CredentialProcess.Worker.Tests/CredentialCreationProcessTypeExecutorTests.cs

```diff
@@ -132,7 +132,7 @@ public async Task InitializeProcess_WithNotExistingProcess_ThrowsNotFoundExcepti
     public async Task ExecuteProcessStep_WithoutRegistrationId_ThrowsUnexpectedConditionException()
     {
         // Act
-        async Task Act() => await _sut.ExecuteProcessStep(ProcessStepTypeId.SIGN_CREDENTIAL, Enumerable.Empty<ProcessStepTypeId>(), CancellationToken.None).ConfigureAwait(false);
+        async Task Act() => await _sut.ExecuteProcessStep(ProcessStepTypeId.SIGN_CREDENTIAL, Enumerable.Empty<ProcessStepTypeId>(), CancellationToken.None);
```

Comment:
async + await is redunant - please remove

Reply:
adjusted as discussed